deadspace/code/modules/clothing/rigs/patron_faction/security_green.dm

```diff
@@ -0,0 +1,46 @@
+// {RedsPro}
+/datum/mod_theme/deadspace/security_greene
+	name = "security greene"
+	desc = "A lightweight and flexible armored rig suit used by earthgov, Urban camo with an inbuilt Bomber Jacket (for the rizz).."
+	default_skin = "greenrig"
+	armor = list(MELEE = 30, BULLET = 30, LASER = 20, ENERGY = 15, BOMB = 45, BIO = 100, FIRE = 100, ACID = 75)
```

Comment:
```suggestion
	armor = list(MELEE = 30, BULLET = 30, LASER = 20, ENERGY = 15, BOMB = 37.5, BIO = 100, FIRE = 60, ACID = 60)
```
Fire should not be 100 unless the RIG has FIRE_PROOF in resistance_flags or the FIRE_SUIT_MAX_TEMP_PROTECT in max_heat_protection_temperature.